tests/components/sonos/test_init.py

```diff
@@ -32,7 +29,7 @@ async def test_creating_entry_sets_up_media_player(hass):
 async def test_configuring_sonos_creates_entry(hass):
     """Test that specifying config will create an entry."""
     with patch(
-        "homeassistant.components.sonos.async_setup_entry", return_value=mock_coro(True)
+        "homeassistant.components.sonos.async_setup_entry",
```

Comment:
The integration setup function should return True on success.
```suggestion
        "homeassistant.components.sonos.async_setup_entry", return_value=True
```